lib/screens/dji_shop/screens/dji_shop_screen/widgets/product_card.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_animation/infrastructure/theme/app_colors.dart';
+import 'package:flutter_animation/infrastructure/theme/app_images.dart';
+import 'package:flutter_animation/infrastructure/theme/theme_extension.dart';
+import 'package:flutter_animation/screens/dji_shop/screens/product_screen/product_screen.dart';
+
+// ---Parameters---
+const _kPadding = 15.0;
+const _kWidthHeight = 300.0;
+
+class ProductCard extends StatelessWidget {
+  const ProductCard({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      padding: const EdgeInsets.only(
+        left: _kPadding,
+        right: _kPadding,
+        top: _kPadding,
+      ),
+      scrollDirection: Axis.horizontal,
+      child: Stack(
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: const [
+              SizedBox(
+                width: _kWidthHeight,
+                child: _CardProduct(),
+              ),
+              // SizedBox(
+              //   width: _kWidthHeight,
+              //   child: _CardProduct(),
+              // ),
+              // SizedBox(
+              //   width: _kWidthHeight,
+              //   child: _CardProduct(),
+              // ),
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _CardProduct extends StatelessWidget {
+  const _CardProduct({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    const nameProduct = Positioned(
+      left: 15,
+      top: 15,
+      child: _CardText(
+        firstWord: 'DJI',
+        secondWord: 'Mavic 3',
+      ),
+    );
+    final priceProduct = Positioned(
+      right: 80,
+      top: 15,
+      child: Text(
+        '\$ 2.049',
+        style: context.theme.textTheme.subtitle1!.copyWith(
+          color: Colors.white,
+          fontWeight: FontWeight.bold,
+        ),
+      ),
+    );
+    final imageProduct = Positioned(
+      top: 15,
+      left: 25,
+      child: InkWell(
+        onTap: () {
+          Navigator.push(
+            context,
+            MaterialPageRoute(
```

Comment:
this `MaterialPageRoute` should be inside `ProductScreen`

Navigation should look like `Navigator.push(context, ProductScreen .getRoute())`